utils/remote-ext-tests/replay-block/src/main.rs

```diff
@@ -0,0 +1,221 @@
+// This file is part of Gear.
+
+// Copyright (C) 2021-2022 Gear Technologies Inc.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+//! Replaying a block against the live chain state
+
+use clap::Parser;
+use codec::Encode;
+use runtime_primitives::Block;
+#[cfg(feature = "always-wasm")]
+use sc_executor::sp_wasm_interface::ExtendedHostFunctions;
+use sc_tracing::logging::LoggerBuilder;
+#[cfg(all(not(feature = "always-wasm"), feature = "gear-native"))]
+use service::GearExecutorDispatch;
+#[cfg(all(not(feature = "always-wasm"), feature = "vara-native"))]
+use service::VaraExecutorDispatch;
+use sp_runtime::{
+    generic::SignedBlock,
+    traits::{Block as BlockT, Header as HeaderT, One},
+};
+use sp_state_machine::ExecutionStrategy;
+use std::fmt::Debug;
+use substrate_rpc_client::{ws_client, ChainApi};
+use util::*;
+
+pub(crate) const LOG_TARGET: &str = "remote-ext::cli";
```

Comment:
too complicated name for target for me, it's not as comfort to use as could be with for example `gear-replay`
And I'd then consider crate renaming into something similar and not such general just because it supports concrete functionality of us as separate and the only one project